CHANGES.rst

```diff
@@ -120,6 +120,7 @@ Accessibility
 ^^^^^^^^^^^^^
 
 - Improve registration form date picker accessibility (:pr:`6371`, thanks :user:`foxbunny`)
+- Improve registration form single choice input accessibility (:pr:`6310`, thanks :user:`foxbunny`)
```

Comment:
Please move the changelog entries to 3.3.4 (a11y section goes between bugfixes and internal changes, since there isn't one for 3.3.4 yet).